src/device-registry/utils/date.js

```diff
@@ -1,7 +1,9 @@
-const { logText, logObject, logElement } = require("./log");
-const isEmpty = require("is-empty");
-const constants = require("../config/constants");
+const { logText, logElement } = require("./log");
 const log4js = require("log4js");
+module.exports = {
+  generateDateFormatWithoutHrs: generateDateFormatWithoutHrs,
```

Comment:
@Baalmart I've added this back as explained, It seems to solve the issue locally.
![image](https://user-images.githubusercontent.com/86492979/193693054-2acddb72-e3fc-4b2d-b186-3a9bac866115.png)

Reply:
Alright, thanks @BenjaminSsempala